README.md

```diff
@@ -81,30 +80,34 @@ sudo zypper rm balena-etcher-electron
 1. Add Etcher rpm repository:
 
    ```sh
-   sudo sudo dnf config-manager --add-repo https://balena.io/etcher/static/etcher-rpm.repo
+   curl -1sLf \
+      'https://dl.cloudsmith.io/public/balena/etcher/setup.rpm.sh' \
+      | sudo -E bash
    ```
 
 2. Update and install:
 
    ```sh
    sudo dnf install -y balena-etcher-electron
+   rm /etc/yum.repos.d/balena-etcher.repo
+   rm /etc/yum.repos.d/balena-etcher-source.repo
```

Comment:
Was adding these lines to the "Update and install:" section a copy&paste mistake?